instrumentation/tomcat-7.0/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/tomcat7/TomcatHandlerTest.groovy

```diff
@@ -56,6 +56,11 @@ class TomcatHandlerTest extends HttpServerTest<Tomcat> implements AgentTestTrait
     endpoint == REDIRECT || endpoint == ERROR
   }
 
+  @Override
+  boolean testErrorBody() {
```

Comment:
@laurit I tried both

```
resp.sendError(serverEndpoint.getStatus(), serverEndpoint.getBody())
resp.setStatus(serverEndpoint.getStatus()
```

but they didn't work. If you think this loosening of the test case can be fixed, please address in a followup :)

Reply:
Instead of `assert response.body().string() == ERROR.body` we could use `assert response.body().string().contains(ERROR.body)` then it would pass with `resp.sendError(serverEndpoint.getStatus(), serverEndpoint.getBody())`. Alternatives would be to figure out how to influence tomcat error page to print just the expected content or remove this assertion completely.

Reply:
> or remove this assertion completely
👍

Reply:
on second thought, maybe the existing check does provide some value...

Reply:
`TomcatServlet3Test` replaces tomcat `ErrorReportValve` copy-pasting that should help